src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -322,11 +325,17 @@ def parse_performance_features(log_lines,
 
 def parse_stats_from_merge_log(log_lines):
   """Extract stats from a log produced by libFuzzer run with -merge=1."""
-  stats = {}
-  for line in log_lines:
-    match = LIBFUZZER_MERGE_LOG_EDGE_COVERAGE_REGEX.match(line)
+  stats = {
+      'edge_coverage': 0,
+      'feature_coverage': 0,
+  }
+
+  # Reverse the list as an optimization. The line of our interest is the last.
+  for line in log_lines[::-1]:
```

Comment:
nit: I believe [::-1] actually creates a new list under the hood (O(n)) so this isn't much of an optimisation :)

Use `reversed(log_lines)` which returns an iterator.

Reply:
I was more worried about applying the regexp to all unnecessary lines, but `reversed` makes a lot of sense here -- thank you, done!